README.md

```diff
@@ -123,6 +123,7 @@ List of options provided by CMake:
 | USE_TSAN | Enable ThreadSanitizer checks | ON/OFF | OFF |
 | USE_MSAN | Enable MemorySanitizer checks | ON/OFF | OFF |
 | USE_VALGRIND | Enable Valgrind instrumentation | ON/OFF | OFF |
+| USE_GCOV | Enable gcov support (Linux only) | ON/OFF | OFF |
```

Comment:
not sure how much we want to care at this point about other compilers, but will this work on gcc or other compilers as well? at least we can describe this somewhere - e.g. in your "example flow", you could add there extra flags `-DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++`

Reply:
It works on gcc and clang